### PR TITLE
introduce the cluster member in the pusher.Client structure

### PR DESCRIPTION
**Pull request by topliceanu**

### CODE REVIEW

**Comment by jpatel531** [comment]
Awesome! LGTM :)

When do you want this merged?

File: request_url.go [hunk 1]
```diff
@@ -43,7 +43,11 @@ func createRequestUrl(method, host, path, key, secret, timestamp string, secure
 	params.Add("auth_signature", authSignature)
 
 	if host == "" {
-		host = "api.pusherapp.com"
+    if cluster != "" {
```
Comment (vivangkumar) [inline request_url.go]: You can use go fmt to format this. The Go standard is to use 8 tabs for indentation. You can just run `go fmt` on the project.

File: request_url.go [hunk 2]
```diff
@@ -43,7 +43,11 @@ func createRequestUrl(method, host, path, key, secret, timestamp string, secure
...
+      host = "api-"+cluster+".pusher.com"
```
Comment (vivangkumar) [inline request_url.go]: You could use `fmt.Sprintf("api-%s.pusher.com", cluster)` - which just formats and returns the string.

**Comment by topliceanu** [comment]
@vivangkumar 10x, didn't know about `go fmt`. 
@jpatel531 probably by the end of the week.

**Pull request merged**